### ISSUE TITLE
Wrong reference in documentation to `YieldFor()`

### ISSUE THREAD

**Issue by taler21** [wxWidgets/wxWidgets#24095]
### Description

`YieldFor()` was added to `wxAppConsoleBase` in d48b06bd9039597545de9a862501d57c5f9ec1ea but moved to `wxEventLoopBase` in dde19c2180ef8d6415af7bb2492bfcb0a2d5c7e4.

Two references in the documentation have apparently not been adjusted and are currently incorrect.

The following patch corrects these references to `wxEventLoopBase::YieldFor()`.

```patch
 docs/doxygen/overviews/thread.h | 2 +-
 interface/wx/app.h              | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/docs/doxygen/overviews/thread.h b/docs/doxygen/overviews/thread.h
index ba78317804..e16bb56b2f 100644
--- a/docs/doxygen/overviews/thread.h
+++ b/docs/doxygen/overviews/thread.h
@@ -36,7 +36,7 @@ To implement non-blocking operations @e without using multiple threads you have
 two possible implementation choices:
 
 - use wxIdleEvent (e.g. to perform a long calculation while updating a progress dialog)
-- do everything at once but call wxWindow::Update() or wxApp::YieldFor(wxEVT_CATEGORY_UI)
+- do everything at once but call wxWindow::Update() or wxEventLoopBase::YieldFor(wxEVT_CATEGORY_UI)
   periodically to update the screen.
 
 However, it is generally much better to run time-consuming tasks in worker threads instead
diff --git a/interface/wx/app.h b/interface/wx/app.h
index b27e504fcb..d5f30778b7 100644
--- a/interface/wx/app.h
+++ b/interface/wx/app.h
@@ -936,7 +936,7 @@ public:
         Works like SafeYield() with @e onlyIfNeeded == @true except that
         it allows the caller to specify a mask of events to be processed.
 
-        See wxAppConsole::YieldFor for more info.
+        See wxEventLoopBase::YieldFor() for more info.
     */
     virtual bool SafeYieldFor(wxWindow *win, long eventsToProcess);
 

```

### Platform and version information

- wxWidgets version  you use: master (8ea22b5e92bf46add0b20059f6e39a938858ff97)

**Comment by vadz** [comment]
Thanks, I'll push this soon (please don't hesitate to open PRs, even for trivial changes like this).

**Comment by taler21** [comment]
Thanks, I think it would make sense to fix it in the 3.2 branch too. TIA!
